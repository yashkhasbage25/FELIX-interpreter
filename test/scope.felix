var name = "Felix";
var foo = fun (){
	var name = "felixlang";
	print("Local Name : ", name);		
};
println("Global Name : ", name);
foo();