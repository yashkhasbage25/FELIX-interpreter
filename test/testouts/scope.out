Global Name : Felix
Local Name : felixlang